apps/finance/app/src/components/DateRange/DateRangeInput.js

```diff
@@ -92,27 +86,69 @@ class DateRangeInput extends React.PureComponent {
     }
   }
 
+  handleApply = e => {
+    e.preventDefault()
+    e.stopPropagation()
```

Comment:
Because of how [handleClickOutside](https://github.com/aragon/aragon-apps/pull/847/files/f7248ae5d6a29a133dbcd4211d52398e3ac5c288#diff-9b38b9af944da3e99573cf03d3e19786R65) is implemented - without the `stopPropagation` the popover would not close.